configuration/initialization.sh

```diff
@@ -50,10 +133,19 @@ echo \
   sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
 sudo apt-get update
 sudo apt-get -y install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
-sudo docker compose up -d
+
+# Check if Port Drayage functionality is enabled
+if [[ $PORT_DRAYAGE_ENABLED == "TRUE" ]]; then
+    sudo docker compose up -d
+elif [[ $PORT_DRAYAGE_ENABLED == "FALSE" ]]; then
```

Comment:
Most of the time, users or at least I will not want port drayage enabled. Won't this only bring up the php container?